docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.9"
    
 services:
   db:
-    image: postgres
+    image: postgres:14
```

Comment:
while we should have a postgres version here, let's use the latest (16 I believe)